### ISSUE TITLE
As a FO, I would like a field that would eliminate options already submitted

### ISSUE THREAD

**Issue by ekellagher** [CuBoulder/express#2002]
Does web express has a choice eliminator option? similar to this: [1]https://chrome.google.com/webstore/detail/choice-eliminator-2/mnhoinjhhhafgieggnhjekliaodnkigj?hl=en-US

[2]	[3]Choice Eliminator 2 - Google Forms add-onchrome.google.comChoice Eliminator will eliminate options from a multiple-choice, list, or checkbox type of question.

________________________________________
Amanda J. McManus
Interactive Media Coordinator
College of Media, Communication and Information
Armory rm. 205
University of Colorado Boulder
Boulder, Colorado 80309
O 303 492 9542
C 303 588 2800
http://colorado.edu/cmci

https://cuboulder.atlassian.net/browse/EXP-9082

**Comment by kreynen** [comment]
![screen shot 2018-03-08 at 1 46 47 pm](https://user-images.githubusercontent.com/385234/37177160-e1b14148-22db-11e8-8818-fe3667d5fefb.png)

Webform Validation's unique validation is just validating that it is unique between 2 fields in the same form, so that doesn't work.
